diff --git a/dyskctl/Makefile b/dyskctl/Makefile
index dd20381..fb8fa19 100644
--- a/dyskctl/Makefile
+++ b/dyskctl/Makefile
@@ -8,7 +8,7 @@ build: ## Builds the `dyskctl` executable
 	$Q cp -Rf cmd vendor/github.com/khenidak/dysk/dyskctl/
 	$Q cp -Rf main.go vendor/github.com/khenidak/dysk/dyskctl/
 	$Q cp -Rf ../pkg/client vendor/github.com/khenidak/dysk/pkg/
-	$Q CGO_ENABLED=0 go build .
+	$Q go build -buildmode=pie -ldflags '-linkmode external -extldflags "-static-pie"' -tags netgo,osusergo .
 
 .PHONY: deps
 deps: ## Runs `dep ensure`
